contracts/evoting/controller/action.go

```diff
@@ -240,97 +248,26 @@ func (a *scenarioTestAction) Execute(ctx node.Context) error {
 
 	// ###################################### CREATE SIMPLE ELECTION ######
 
-	fmt.Fprintln(ctx.Out, "Create election")
-
-	// Define the configuration
-	configuration := fake.BasicConfiguration
-
-	createSimpleElectionRequest := ptypes.CreateElectionRequest{
-		Configuration: configuration,
-		AdminID:       "adminId",
-	}
-
-	signed, err := createSignedRequest(secret, createSimpleElectionRequest)
+	err,electionID,election,electionIDBuf :=setupSimpleElection(ctx,secret,proxyAddr1,serdecontext,electionFac,service)
 	if err != nil {
-		return createSignedErr(err)
+		return err
```

Comment:
We usually try to add a small message to explain an error and make the traceback easier to follow.
```suggestion
		return xerrors.Errorf("failed to setup simple election: %v", err)
```